### DISCUSSION TITLE
I dont think the stats command is broken or even with little rate the connection gets off

### DISCUSSION THREAD

**Discussion by MR-pentestGuy** [projectdiscovery/nuclei#1020] [General]
**Describe the bug**
stats command not working properly 

**Nuclei version**
2.5.1 (latest)

**Screenshot of the error or bug**
![image](https://user-images.githubusercontent.com/71591447/133201735-a0f536c4-ce83-47c9-8d09-7476672a4209.png)

**Comment by ehsandeep** [comment]
@MR-pentestGuy can you please be specific about issue details? for example

- what is the issue
- what was expected and not working accordingly
- cmd to reproduce

**Reply by MR-pentestGuy** [reply]
the issue is it does not complete 100 as stats shows us how much of the tasks is done but here it shows 30 is done and then it quits

cmd i have used 
nuclei -l http_subs.txt -t ~/nuclei-templates/ -include-tags misc,fuzz -c 10 -headless -impact medium -stats -o scanner/nuclei/medium.txt
